features/integration/applications.feature

```diff
@@ -139,8 +139,7 @@ Feature: Applications
       And I wait for the transaction to be confirmed.
       Then according to "algod", the current application should have the following boxes "".
 
-    @applications.boxes_indexer_confirmed
-    Scenario: Exercise indexer after a slew of box creations
```

Comment:
You don't need the `@application.boxes` tag here?

Reply:
using it now!